### DISCUSSION TITLE
[p. 291: 실습코드 5-5. ApiService에서 catch & redirect]

### DISCUSSION THREAD

**Discussion by sjsage522** [fsoftwareengineer/todo-application#10] [질문과 답변]
fetch 호출후 catch 부분
...
        .catch((error) => {
            console.log(error.status);
            if (error.status === 403) {
                window.location.href = "/login"; //redirect
            }
            return Promise.reject(json); /* json 부분 빨간줄 */
        });

json -> error 로 수정하면 컴파일에러가 해결됩니다.

**Comment by leeyou34** [comment]
오우 완전 감사해요! 

return Promise.reject(json);  ->  return Promise.reject(error); 로 바꾸니 잘 됩니당!
